freqtrade/freqtradebot.py

```diff
@@ -479,12 +462,33 @@ def process_maybe_execute_buys(self) -> None:
         """
         Tries to execute buy orders for trades in a safe way
         """
-        try:
-            # Create entity and execute trade
-            if not self.create_trades():
-                logger.debug('Found no buy signals for whitelisted currencies. Trying again...')
-        except DependencyException as exception:
-            logger.warning('Unable to create trade: %s', exception)
+        result = False
+
+        whitelist = copy.deepcopy(self.active_pair_whitelist)
+        if not whitelist:
+            logger.info("Active pair whitelist is empty.")
+        else:
```

Comment:
could you make this change by yourself (either here or in a new PR, whatever suites you better)?so that I could review this in the code, not in the textual description...